doc/prometheusrules/apicast.yaml

```diff
@@ -53,7 +53,7 @@ spec:
             LABELS: {{ $labels }}
         sop_url: https://github.com/3scale/3scale-Operations/blob/master/sops/alerts/apicast_apicast_latency.adoc
         summary: APICast latency high (instance {{ $labels.instance }})
-      expr: histogram_quantile(0.99, sum(rate(total_response_time_seconds_bucket{namespace='__NAMESPACE__',}[30m]))
```

Comment:
this file is generated automatically, the source of truth for this file is `apicast_monitoring.go`

Reply:
Thank you for pointing that out.